spec/datadog/tracing/contrib/sidekiq/distributed_tracing_spec.rb

```diff
@@ -0,0 +1,119 @@
+# typed: ignore
```

Comment:
I think you can write just two tests: one with `distributed_tracing: true` and another with it `false`.

Then the test setup is like:
1. Start a span in the test run. Set a non-default sampling priority, origin, and a dummy `_dd.p.*` tag in the active trace.
1. Run `EmptyWorker.perform_async` with the previous span still active.
1. Check that you have two spans, where the `perform_async` span is a child of the span in `1`.
2. Ensure both spans have the same sampling priority, origin, and `_dd.p.*` tags.